node_account.proto

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018 REMME
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+syntax = "proto3";
+
+message NodeAccount {
+    bool is_masternode = 1;
+    bool is_masternode_closed = 3;
+
+    uint64 balance = 4;
+
+    message Reputational {
```

Comment:
This message is unused.  If you intended to use it as a field, you need to instantiate it.

Reply:
Something like this `repeated Reputational reputation = 3;`?

Reply:
Why `repeated`? If this message is per node, then just a single field.